importScripts/sampleRecords/library.source.json

```diff
@@ -0,0 +1,738 @@
+[{
+  "PublicationID": "10",
+  "Title": "The 'Caroline' ; 150 years in Western Australia of the Gee and Truslove families 1829-1979",
+  "LegacyID": "10",
+  "Series": "0",
+  "Authors": "Aamot, Iris",
+  "LastName": "Aamot",
+  "FirstNames": "Iris",
+  "HomeLocation": "LIBRARY",
+  "LastEditBy": "LanK",
+  "LastEditDate": "2004-03-22 00:00:00",
+  "LibraryNo": "AAM",
+  "PublicationType": "Book",
+  "PublicationYear": "1979",
+  "Publisher": "Author",
+  "WherePublished": "Mandurah",
+  "Subject": "Gee family - History; Truslove family - History.",
+  "Illustrations": "1",
+  "Maps": "0",
+  "Marks": "vi, 50p. illus., photos.",
+  "Pages": "50",
+  "Rare": "0",
+  "Restrictions": "NOT FOR LOAN",
+  "DateDue": "2002-02-14",
+  "DateOut": "2002-01-17",
+  "Borrower": "NOT FOR LOAN",
+  "Value": "0"
```

Comment:
Only two records have a value other than '0' and this is not included in the library UI so not migrating this field. Will clarify with the client.